pkgs/by-name/eq/equicord/package.nix

```
@@ -68,7 +68,7 @@ stdenv.mkDerivation (finalAttrs: {
     description = "Other cutest Discord client mod";
     homepage = "https://github.com/Equicord/Equicord";
     license = lib.licenses.gpl3Only;
-    platforms = lib.platforms.linux;
+    platforms = lib.platforms.all;
```

```suggestion
    platforms = lib.platforms.unix;
```
darwin+linux is unix, not all.